cypress/component/rich-text/RichTextEditor.Tables.spec.ts

```diff
@@ -173,7 +105,9 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
       );
     });
 
-    it('does not delete table header cells when selecting the whole header row', () => {
+    // Skipping this test because of weird Plate behavior. Will describe it in a follow up PR.
+    // eslint-disable-next-line mocha/no-skipped-tests
+    it.skip('does not delete table header cells when selecting the whole header row', () => {
```

Comment:
❗ We are skipping quite a few tests. Why not keep them as e2e instead?

Reply:
I would honestly prefer to get rid of e2e and address these tests with follow up PRs.

Reply:
Would the follow up happen right away or would be in a different ticket?

Reply:
In a different ticket (already created). We can add it to the next sprint.